uaclient/entitlements/base.py

```diff
@@ -281,6 +281,7 @@ def __init__(
         allow_beta: bool = False,
         called_name: str = "",
         access_only: bool = False,
+        purge: bool = False,
```

Comment:
Can't this be an attribute we use when calling the `disable` method ?

Reply:
I guess we could - I am setting it in the class to mimic what was done with `assume_yes` and `access_only`, so for consistency I defined it similarly

Reply:
The others were added as instance fields because they were needed in other methods where we couldn't easily pass them from the enable function otherwise.
I don't think this option is needed in any calls outside `disable`, but also I don't think it hurts to follow the patterns of the others, and we may need it somewhere else in the future

Reply:
we need this to avoid the pre-disablement message on FIPS at least - so I think following the pattern is the way to go